test/integration.js.snap

```diff
@@ -8,7 +8,7 @@ File               | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
 -------------------|---------|----------|---------|---------|-------------------
 All files          |   86.21 |    91.67 |   66.67 |   86.21 |                   
  async.js          |     100 |      100 |     100 |     100 |                   
- c8-ignore-next.js |   90.91 |      100 |     100 |   90.91 | 21-22             
```

Comment:
Outside of cobertura the only regressions are the change from `\d+-\d+` to `\d+,\d+` which seems like the *correct* value for 1 line ranges?